src/modules/75-cd/components/PipelineSteps/K8sServiceSpec/ArtifactSource/ECRArtifactSource/ECRArtifactSource.tsx

```diff
@@ -62,11 +63,14 @@ const Content = (props: ECRRenderContent): JSX.Element => {
   const { getString } = useStrings()
   const isPropagatedStage = path?.includes('serviceConfig.stageOverrides')
   const { expressions } = useVariablesExpression()
-
+  const [lastQueryData, setLastQueryData] = React.useState<{ imagePath: string; region: any }>({
```

Comment:
Codacy has a fix for the issue: [Unexpected any. Specify a different type.](https://app.codacy.com/gh/harness/harness-core-ui/pullRequest?prid=9254512)
```suggestion
  const [lastQueryData, setLastQueryData] = React.useState<{ imagePath: string; region: unknown }>({
```